src/lib/adblock/adblockmanager.cpp

```diff
@@ -154,10 +154,14 @@ bool AdBlockManager::addSubscriptionFromUrl(const QUrl &url)
 
     for (int i = 0; i < queryItems.count(); ++i) {
         QPair<QString, QString> pair = queryItems.at(i);
-        if (pair.first == QL1S("location"))
-            subscriptionUrl = pair.second;
-        else if (pair.first == QL1S("title"))
-            subscriptionTitle = pair.second;
+        if (pair.first == QL1S("location")) {
+            QString temp = pair.second;
+            subscriptionUrl = QUrl(temp).path();
```

Comment:
It works, but maybe the correct fix is just `QUrlQuery::queryItems(QUrl::FullyDecoded))`